src/main/java/book/store/onlinebookstore/dto/categorydto/CreateRequestCategoryDto.java

```diff
@@ -0,0 +1,13 @@
+package book.store.onlinebookstore.dto.categorydto;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+public record CreateRequestCategoryDto(
+        Long id,
+        @NotNull
+        @NotEmpty
```

Comment:
According to NotEmpty annotation javadoc:
"The annotated element must not be null nor empty."

This means NotNull is not required when we have NotEmpty

```suggestion
        @NotEmpty
```